dev/manual_tests/lib/raw_keyboard.dart

```diff
@@ -50,9 +51,16 @@ class _HardwareKeyDemoState extends State<RawKeyboardDemo> {
     return index == -1 ? name : name.substring(index + 1);
   }
 
+  @override
+  void initState() {
+    super.initState();
+    _focusAttachment = _focusNode.attach(context);
+  }
+
   @override
   Widget build(BuildContext context) {
     final TextTheme textTheme = Theme.of(context).textTheme;
+    _focusAttachment.reparent(Focus.of(context));
     return RawKeyboardListener(
       focusNode: _focusNode,
       onKey: _handleKeyEvent,
```

Comment:
seems like there's a missing call to `detach` here?

Reply:
Well, the `dispose` has an implicit `detach`, and I don't need to detach to reparent: it will only actually reparent if it's attached.

Reply:
I also don't see a dispose?